score/scripts/download_cbg.py

```diff
@@ -0,0 +1,106 @@
+import csv
+import requests
+import zipfile
+import os
+import json
+
+with requests.Session() as s:
```

Comment:
I also thing this could be revisited later -- we/the user won't be downloading census data often. This script would be typically be run the very first time you setup the repo locally.